src/main/webapp/script.js

```diff
@@ -99,6 +113,35 @@ function logAddressInput(){
                     `${streetNumber} ${route} ${city} ${state} ${zipcode} ${country}`);
 }
 
+/**
+ * Redirect to a given URL while maintaining the query string
+ * and tracking the name, ID of the chosen state election
+ */
+function redirectToStateElectionPage(electionId){
+  let electionName = document.getElementById('state-election-name').innerText;
+  goToElectionPage(electionName, electionId);
+}
+
+/**
+ * Redirect to a given URL while maintaining the query string
+ * and tracking the name, ID of the chosen national election
+ */
+function redirectToNationalElectionPage(electionId){
+  let electionName = document.getElementById('national-election-name').innerText;
+  goToElectionPage(electionName, electionId);
+}
+
+/**
+ * Redirect to third page and update the query string with election info.
+ */
+function goToElectionPage(electionName, electionId){
+  addQueryParameter("electionName", electionName);
+  addQueryParameter("electionId", electionId);
+
+  var query = window.location.search;
+  window.location.href = "electionInfo.html" + query;
```

Comment:
If we're moving to a new webpage, can't the server fill out the election name?  As opposed to having electionName passed in to this method.

Reply:
How would the server do that?

Reply:
You probably will want to change electionInfo.html to be dynamically populated.  There are a lot of different ways to do this, using a templating engine is probably the way to go (similar to handlebars.js but done on the server side).

I would recommend using jsp since we're using java servlets.

https://happycoding.io/tutorials/java-server/jsp

I think this tutorial should have most of if not all the information to get started.

Reply:
Got it -- I had used this method because I was most familiar with it and wanted to complete this task quickly for the sake the MVP, but can switch over to a java servlet soon (later this week or early next week).

Reply:
Can you just leave TODO here then?

Reply:
Definitely -- will do right now. Thanks!